src/index.ts

```diff
@@ -34,6 +36,13 @@ export async function run(): Promise<void> {
     // remove white spaces (if any) in archs input
     archs = archs.replace(/\s+/g, "");
 
+    // To solve https://github.com/redhat-actions/buildah-build/issues/45
+    const homedir = os.homedir();
+    const storageFilePaths: string[] = [ path.join(homedir, ".config/containers/storage.conf"),
+        "/etc/containers/storage.conf" ];
+    // If overlay is used as storage driver then set "fuse-overlayfs as mount program"
+    checkStorageDriver(storageFilePaths);
```

Comment:
wrap this in a try/catch with an error log, but the action should not fail if this fails (though I suppose it may fail at a later step due to the issue we are trying to work around)